src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -139,50 +139,12 @@ private JObject LoadHostConfigurationFile()
                 string sanitizedJson = SanitizeHostJson(hostConfigObject);
                 string readFileMessage = $"Host configuration file read:{NewLine}{sanitizedJson}";
 
-                // TODO: DI (FACAVAL) See method comments in ScriptHost
-                //ApplyConfiguration(hostConfigObject, ScriptConfig, _startupLogger);
-
-                // TODO: DI (FACAVAL) Review - this should likely move and not be just config
-                //if (_settingsManager.FileSystemIsReadOnly)
-                //{
-                //    // we're in read-only mode so source files can't change
-                //    ScriptConfig.FileWatchingEnabled = false;
-                //}
-
-                // now the configuration has been read and applied re-create the logger
-                // factory and loggers ensuring that filters and settings have been applied
-                // TODO: DI (FACAVAL) TODO
-                //ConfigureLoggerFactory(recreate: true);
```

Comment:
All logger configuration is handled by the logging changes binding to the appropriate section.